src/appleseed/renderer/modeling/frame/frame.h

```diff
@@ -107,9 +108,15 @@ class APPLESEED_DLLSYMBOL Frame
     // Access the AOV images.
     ImageStack& aov_images() const;
 
+    // Access extra AOV indexs.
+    std::vector<size_t>& extra_aov_indexs() const;
+
     // Create an extra AOV image if it does not exist.
     size_t create_extra_aov_image(const char* name) const;
 
+    // Check if extra AOVs are enabled.
+    bool has_extra_aov() const;
```

Comment:
Is this method really needed? It seems completely redundant with `extra_aov_indexes()`.